services/apis/indexd/indexdQuestions.js

```diff
@@ -38,9 +38,33 @@ module.exports = {
    * Asserts a record was successfully deleted from indexd
    * @param {Object} ffileNode
```

Comment:
```suggestion
   * @param {Object} fileNode
```
I may be looking at docstrings too much 😆 but can you add parameter `res` here (mostly to make it clear that it's the entire response)